docs/installation/linux/ubuntulinux.md

```diff
@@ -82,10 +82,6 @@ packages from the new repository:
 
             deb https://apt.dockerproject.org/repo ubuntu-trusty main
 
-    - Ubuntu Wily 15.10
```

Comment:
You forgot one at the top;
https://github.com/docker/docker/pull/27042/files#diff-26ed3ddfd9bfa4b4334997b3d9bcad44R18